.github/workflows/deploy-virus-scanner-legacy-staging.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy Legacy Virus Scanner to new OGP AWS environment
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - staging-alt3
+  # For legacy virus scanner: schedule builds for 12:00AM GMT+8 everyday to get latest ClamAV virus definitions
+  schedule:
+    - cron: '0 16 * * *'
+
+
+jobs:
+  deploy-scanner:
+    name: Deploy Scanner
+    uses: ./.github/workflows/aws-deploy-scanner-legacy.yml
+    with:
+      checkoutBranch: staging-alt3
+      environment: stg-alt3
+      provisionedConcurrency: 1
+    secrets: 
+      cicd-role: ${{ secrets.STG_ALT3_AWS_CI_ROLE_TO_ASSUME }}
```

Comment:
I dont' see `STG_ALT3_AWS_CI_ROLE_TO_ASSUME` in secrets 🤔 is this supposed to be `AWS_CI_ROLE_TO_ASSUME`?

Reply:
2674c4f